ratpack/src/test/groovy/com/baeldung/RatpackGroovySpec.groovy

```diff
@@ -0,0 +1,47 @@
+package com.baeldung;
+
+import ratpack.groovy.Groovy
+import ratpack.groovy.test.GroovyRatpackMainApplicationUnderTest;
+import ratpack.test.http.TestHttpClient;
+import ratpack.test.ServerBackedApplicationUnderTest;
+import org.junit.Test;
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import ratpack.test.MainClassApplicationUnderTest
+
+class RatpackGroovySpec {
+
+    ServerBackedApplicationUnderTest aut = new MainClassApplicationUnderTest(RatpackGroovyApp.class)
```

Comment:
let's use a full name for the application under test - like `application` or even `app` - `aut` is very cryptic.

Reply:
agreed